starter/js/atm.js

```diff
@@ -1,43 +1,76 @@
 //Begin with the document ready function
+$(document).ready(function() {
 
-    //Checking account deposit function
 
-      //On click of the depositChecking button
+  var checkingBalance = $('#checkingBalance');
+  var amountChecking = $('#amountChecking');
+  var withdrawChecking = $('#withdrawChecking');
 
-        //Get value from the amountChecking input field
+  //Checking account deposit function
+  $('#depositChecking').on('click', function() {
+    var totalAmountChecking = parseInt(amountChecking.val());
+    var totalCheckingBalance = parseInt(checkingBalance.text());
 
-        //Take that value and add it to the existing value in the checkingBalance div
+    // console.log(totalCheckingBalance);
+    var checkingTotal = (totalCheckingBalance + totalAmountChecking);
+    checkingBalance.text(checkingTotal);
+  });
+  //Checking account withdrawl function
+  //On click of the withdrawChecking button
+  //Get value from the amountChecking input field
+  // If that value is greater than the value in the account ignore that action
+  // In other words if this would put the account into a negative balance do not allow it
+  var withdrawSavings = $('#withdrawSavings')
```

Comment:
Kind of makes sense to declare all these variables in one place. You have them scattered throughout your code.